pom.xml

```diff
@@ -160,6 +166,11 @@
       <groupId>org.apache.poi</groupId>
       <version>${poi.version}</version>
     </dependency>
+    <dependency>
```

Comment:
let's definitely remove this. This only resolves because authorization-api lives locally in your .m2 directory. It won't be able to download this from maven central at which all dependencies live